### DISCUSSION TITLE
Welcome to opics Discussions!

### DISCUSSION THREAD

**Discussion by lukasc-ubc** [SiEPIC/opics#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EmaOPTIUS** [comment]
Hi, 
I tried to play a little with opics - great initiative! It's always cool to have some scripts to double-check simulations, that's how I wanted to use it today.
I have an issue loading s-parameter of the example files. 
Typing this
```
path = 'C:\\Users\\emanuel.peinke\\AppData\\Local\\Programs\\Spyder\\pkgs\\opics\\libs\\ebeam\\data\\ebeam_dc_halfring_straight'
name = 'te_ebeam_dc_halfring_straight_gap=100nm_radius=10um_width=500nm_thickness=220nm_CoupleLength=0um_mc.xml'
dc_halfring.load_sparameters(path, name)  
```
I get the following error:
```
File "C:\Users\emanuel.peinke\AppData\Local\Programs\Spyder\pkgs\opics\utils.py", line 158, in LUT_reader
xml = ET.parse(filedir/ lutfilename)
TypeError: unsupported operand type(s) for /: 'str' and 'str'
```
How should I load the example files properly?

**Reply by jaspreetj** [reply]
Hi Emanuel,

I'm happy that you are finding OPICS useful. :) 

As for the error: this is because the filedir should be a `Path` object. Try the following:
```python3
from pathlib import Path
path = Path(r'C:\\Users\\emanuel.peinke\\AppData\\Local\\Programs\\Spyder\\pkgs\\opics\\libs\\ebeam\\data\\ebeam_dc_halfring_straight')
name = r'te_ebeam_dc_halfring_straight_gap=100nm_radius=10um_width=500nm_thickness=220nm_CoupleLength=0um_mc.xml'
dc_halfring.load_sparameters(path, name)
```
